### ISSUE TITLE
Is it possible to enhance this package with slave/server

### ISSUE THREAD

**Issue by biancode** [yaacov/node-modbus-serial#44]
Is it possible to enhance this package with slave/server, please?

http://www.rtaautomation.com/technologies/modbus-tcpip/

**Comment by yaacov** [comment]
> Is it possible to enhance this package with slave/server

a. yes :-)
b. what is the use case ?
c here is mock server, is this what you have in mind ?
```
// create an empty modbus server
var ModbusRTU = require("modbus-serial");
var server = new ModbusRTU();

// open connection to a tcp line
// and register some response functions 
client.serveTCP({url: "0.0.0.0", port: 502} , { 
    readInputRegisters: function(address, length) { return [42; ]; },
    writeCoil: function(address, state) { console.log("writing coil"); } 
});

```

**Comment by biancode** [comment]
a. great!
b. https://github.com/biancode/node-red-contrib-modbus/blob/master/src/modbus-server.js
want to use only modbus-serial in node-red package
c. looks good and also see https://github.com/biancode/node-red-contrib-modbus/blob/master/src/modbus-server.js from line 111

**Comment by yaacov** [comment]
fixed by #52

see example:
https://github.com/yaacov/node-modbus-serial/blob/master/examples/server.js

**Comment by biancode** [comment]
cool - I'll start testing next week
